src/app/shared/components/prompt/prompt-fields.ts

```diff
@@ -142,164 +160,41 @@ export function INVITATION_FIELDS(initialEmail?: string): PromptField[] {
       config: {
         autocapitalize: 'on',
         autocorrect: 'off',
-        autocomplete: 'off'
-      }
+        autocomplete: 'off',
+      },
     },
     RELATIONSHIP_FIELD,
   ];
   return fields;
 }
 
-export const CREDIT_CARD_FIELDS: PromptField[] = [
-  {
-    fieldName: 'cardNumber',
-    placeholder: 'Card number',
-    type: 'tel',
-    validators: [Validators.required],
-    config: {
-      autocomplete: 'cc-number',
-      autocorrect: 'off',
-      autocapitalize: 'off'
-    }
-  },
-  {
-    fieldName: 'cardCvc',
-    placeholder: 'Security code',
-    type: 'tel',
-    validators: [Validators.required],
-    config: {
-      autocomplete: 'cc-csc',
-      autocorrect: 'off',
-      autocapitalize: 'off'
-    }
-  },
-  {
-    fieldName: 'cardExpMonth',
-    placeholder: 'Expiration month',
-    type: 'select',
-    validators: [Validators.required],
-    config: {
-      autocomplete: 'cc-exp-month',
-      autocorrect: 'off',
-      autocapitalize: 'off'
-    },
-    selectOptions: expMonths.map((month) => {
```

Comment:
Looks like this was the last reference to `expMonths`, so let's delete that array, too!